cfgov/v1/jinja2/v1/includes/molecules/translation-links.html

```diff
@@ -36,17 +36,23 @@
 {% endif -%}
 
 {%- if value.links and value.links|length > 1 %}
-<div class="m-translation-links">
-{%- for link in value.links %}
-    {% set render_link = value.language and link.language != value.language %}
-    {% if render_link -%}
-        <a href="{{ link.href }}" hreflang="{{ link.language }}">
-    {%- endif %}
-    {%- language link.language %}{{- _( link.text ) -}}{% endlanguage -%}
-    {% if render_link %}</a>{% endif %}
-    {% if not loop.last %}|{% endif %}
-{% endfor -%}
-</div>
+<ul dir="ltr" class="m-translation-links">
+    {%- for link in value.links %}
+        {% set render_link = value.language and link.language != value.language %}
```

Comment:
```suggestion
        {%- set render_link = value.language and link.language != value.language %}
```

Some Jinja changes to make the markup a little nicer:

<table>
<tr><td>Before</td><td>After</td></tr>
<tr><td>

```html
<ul dir="ltr" class="m-translation-links">
        
    <li>
    English
    </li>

    
    <li>
    <a href="/about-us/blog/translated-page-spanish/"
           lang="es"
           hreflang="es"
           translate="no">Español
        </a>
    
    </li>

    
    <li>
    <a href="/about-us/blog/translated-page-chinese/"
           lang="zh-Hant"
           hreflang="zh-Hant"
           translate="no">中文
        </a>
    
    </li>
```

</td><td>

```html
<ul dir="ltr" class="m-translation-links">
        <li>English</li>
    
        <li>
            <a href="/about-us/blog/translated-page-spanish/"
               lang="es"
               hreflang="es"
               translate="no">Español
            </a>
        </li>
    
        <li>
            <a href="/about-us/blog/translated-page-chinese/"
               lang="zh-Hant"
               hreflang="zh-Hant"
               translate="no">中文
            </a>
        </li>
```

</td></tr></table>